chain/db/Delegation.go

```diff
@@ -0,0 +1,34 @@
+package db
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+)
+
+func (b *BandDB) AddDelegation(
```

Comment:
This is still not correct. You will need to handle slashing, inflation, etc to make delegate works. Let's comment out all the code for now.